cmds/uinit/uinit.go

```diff
@@ -14,27 +14,68 @@ import (
 
 var (
 	commands = []string{
-		"/buildbin/date",
-		"/buildbin/ls -l",
+		"/bin/bash",
 	}
 )
 
-func main() {
-	for _, line := range commands {
-		log.Printf("Executing Command: %v", line)
-		cmdSplit := strings.Split(line, " ")
-		if len(cmdSplit) == 0 {
-			continue
-		}
-
-		cmd := exec.Command(cmdSplit[0], cmdSplit[1:]...)
-		cmd.Stdin = os.Stdin
-		cmd.Stderr = os.Stderr
-		cmd.Stdout = os.Stdout
-		if err := cmd.Run(); err != nil {
-			log.Print(err)
-		}
+func start() {
+	flag.Usage = usage
+	// This getpid adds a bit of cost to each invocation (not much really)
+	// but it allows us to merge init and sh. The 600K we save is worth it.
+	// Figure out which init to run. We must always do this.
+
+	// log.Printf("init: os is %v, initMap %v", filepath.Base(os.Args[0]), initMap)
+	// we use filepath.Base in case they type something like ./cmd
 
+	log.Printf("init: Making mount directory")
+
+	if err := syscall.Mkdir("/mnt", 0777); err != nil {
```

Comment:
it would be nicer if you could find a way to use the package in uroot/root.go, since that is used by the other tools, and is table driven. take a look and let me know what you think. init.go uses it and you can use your own tables with it.